app/_how-tos/allow-multiple-authentication.md

```diff
@@ -45,8 +73,139 @@ cleanup:
       icon_url: /assets/icons/gateway.svg
 ---
 
-@todo
+## 1. Create an ID
+
+Create a UUID:
```

Comment:
```suggestion
You can use multiple authentication plugins with an anonymous Consumer to give clients multiple options for authentication. The anonymous Consumer acts as a fallback to catch all other unauthorized requests.

First, you must create a UUID for the anonymous Consumer. This is used when you configure the authentication plugins.
```
Added the "intro" content from the TLDR here. I also wanted to know why we needed a UUID, so I added that here as well, not sure if that's correct.